docs/install/install-kn.md

```diff
@@ -13,13 +13,33 @@ To install the `kn` Client, you must download the executable binary for your sys
 You must place the executable binary in your system path, and make sure that it is executable.
 
 ## Install `kn` using Go
-1. Check out the [Client repository](https://github.com/knative/client).
-1. Run the command:
+1. Build `kn` requires Go v1.13 or newer. Please install [go](https://golang.org/doc/install) environment firstly.
+1. Download the [Client repository](https://github.com/knative/client).
+  ```bash
+  cd $GOPATH/src/knative-dev
+  git clone https://github.com/knative/client.git
+  cd client/
+  ```
+1. Build an executable binary:
   ```bash
   go install ./cmd/kn
   ```
+1. Move `kn` into your system path, and verify whether kn is working properly. For example:
+  ```bash
+  cp $GOPATH/bin/kn /bin/
+  kn version
+  ```
 
 ## `kn` container images
 The `kn` container images are available here:
 - [Nightly container image](https://gcr.io/knative-nightly/knative.dev/client/cmd/kn)
 - [Latest release](https://gcr.io/knative-releases/knative.dev/client/cmd/kn)
+
+You can run `kn` from container. For example:
+  ```bash
+  docker pull https://gcr.io/knative-releases/knative.dev/client/cmd/kn
```

Comment:
You don't need to pull in advance, that will automatically happen when you call `docker run`

Reply:
:+1: 

not `https://` prefix for image references